web3.js/src/util/send-and-confirm-transaction.ts

```diff
@@ -2,7 +2,26 @@ import {Connection} from '../connection';
 import {Transaction} from '../transaction';
 import type {ConfirmOptions} from '../connection';
 import type {Signer} from '../keypair';
-import type {TransactionSignature} from '../transaction';
+import type {
+  TransactionSignature,
+} from '../transaction';
+
+class TransactionExpiredBlockheightExceededError extends Error {
+  signature: string;
+
+  constructor(signature: string) {
+    super(`Signature ${signature} has expired.`);
+    this.signature = signature;
+  }
+}
+
+Object.defineProperty(
+  TransactionExpiredBlockheightExceededError.prototype,
+  'name',
+  {
+    value: 'TransactionExpiredBlockheightExceededError',
+  },
+);
```

Comment:
These should:

* be thrown from `confirmTransaction` when timeouts happen.
* live in `transaction.ts`, or their own file.

Reply:
Moved to `transaction.ts`